### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_WAT: Color Contrast is less than 4.5:1 for Menus(File, Edit, View....etc).

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4000]
**Environment Details:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page     opens with menubar.
3.  Open WAT Tool.
4. Open Color Contrast Analyser.
5. Verify Color Contrast for Menus(File, Edit, View....etc).

**Actual:**
Color Contrast is less than 4.5:1 for Menus(File, Edit, View....etc).

**Expected:**
Color Contrast should be >= 4.5:1 for Menus(File, Edit, View....etc).

**User Impact:**
If ColorContrast is <4.5:1 than Low Vision Users face difficulty to See the Menus.

**Attachment:**
![a11y_azurenotebooks_jupyter_menubar_wat](https://user-images.githubusercontent.com/39204909/45815023-1d563b00-bcf5-11e8-8b31-88fe08d17dd9.png)

**Comment by bhansa** [comment]
Hi, I have made a little change to solve color contrast issue, here is the screenshot:

![image](https://user-images.githubusercontent.com/9723884/46608494-91307a00-cb22-11e8-8d00-45d569aab286.png)

Let me know if I can go ahead and commit the code :)
